<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小梨子工具箱</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PDFToWordBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF To Word</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>30</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>小梨子工具箱</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PDFToWordLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF转Word工具</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rsLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="csLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="rsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rs</string>
    </property>
   </widget>
   <widget class="QLabel" name="csLable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>cs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>小梨子</string>
    </property>
   </widget>
   <widget class="QLabel" name="targetLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Target</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchExcelBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel 查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="excelSearchLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel查询工具</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="runOutPutTextBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="runOutputLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>运行输出</string>
    </property>
   </widget>
   <widget class="QLabel" name="reLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>re</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ceLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="ceLable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ce</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>联系方式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoContentLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="excelStatisticsLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel统计工具</string>
    </property>
   </widget>
   <widget class="QLabel" name="excelStatisticsOriginLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>原始表格</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reStaLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="csStaLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="rsStaLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rs</string>
    </property>
   </widget>
   <widget class="QLabel" name="csStaLable">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>cs</string>
    </property>
   </widget>
   <widget class="QLabel" name="ceStaLable">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ce</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ceStaLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rsStaLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="reStaLable">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>re</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colStaSplit1LineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="colStaSplit2Label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>c-s-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="colStaSplit1Lable">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>c-s-1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colStaSplit2LineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="excelStatisticsTargetLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>目标表格</string>
    </property>
   </widget>
   <widget class="QLabel" name="staPos1Lable">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>p1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="staPos1LineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="staPos2LineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="staPos2Lable">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>p2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StatisticsBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel 统计</string>
    </property>
   </widget>
   <widget class="QLabel" name="reStaTarLable">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>re</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reStaTarLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>66</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rsStaTarLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="rsStaTarLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rs</string>
    </property>
   </widget>
  </widget>
  <action name="action123">
   <property name="text">
    <string>关于 </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
